browser-test/src/admin/admin_question_references.test.ts

```diff
@@ -121,5 +120,52 @@ test.describe(
         await validateScreenshot(page, 'question-program-modal')
       })
     })
+
+    test('shows results for referencing disabled program', async ({
```

Comment:
suggestion (non-blocking): Edit the test name to explain what the results shown should be. Something like "shows disabled programs listed separately"

Reply:
updated the test name. Let me know what you think.